activiti-spring-process-extensions/src/main/java/org/activiti/spring/process/model/VariableDefinition.java

```diff
@@ -13,49 +13,14 @@
 
 package org.activiti.spring.process.model;
 
-public class VariableDefinition {
+public class VariableDefinition extends org.activiti.core.common.model.connector.VariableDefinition {
```

Comment:
see above comment about moving VariableDefinition to a common module. (will also need to add a value property)

Reply:
@balsarori then I guess we can have a single `VariableDefinition`, no reason to get it in two different places, right?

Reply:
yes we need to added ``value``